### PR TITLE
Simplify `.gitattributes` in Submitting to the Asset Library

### PR DESCRIPTION
**Pull request by nathanfranke**
Context: https://github.com/godotengine/godot-proposals/issues/3648

### CODE REVIEW

**Review by aaronfranke — APPROVED** [review]

**Comment by YuriSizov** [comment]
Asset is not necessarily an addon, so this is not a good change.

**Comment by aaronfranke** [comment]
@YuriSizov The existing `.gitattributes` on the page is already an example for an addon, since it prevents `project.godot` from being exported. Maybe this should be made more clear, or we should do better at explaining that this example is for addons, and for projects they should not include any of the `export-ignore` lines.

**Comment by YuriSizov** [comment]
@aaronfranke Yes, I agree that the preceding text puts this example in the context of a plugin/addon. In that sense, this change is fine. But I also like that the current example gives an extra reminder that you probably don't want to include project files, and screenshots, and git metafiles. An asset can be simply a collection of reusable scenes, like a custom control or a textured/scripted game object. So it's not an addon, and it's not a template project.

But either way, that's just my thoughts. I'm not going to block this PR if no adjustments are made.

**Comment by gumaciel** [comment]
I think this change is valid.

I was thinking of having two types of `.gitattributes` to ignore some files when doing the `.zip`: One for Addons and one for Projects, but I don't know what `.gitattributes` for Projects would be like

**Comment by aaronfranke** [comment]
@gumaciel For projects, the `.gitattributes` file should not export-ignore anything, so the proper file looks like this:

```gitattributes
# Normalize EOL for all files that Git considers text files.
* text=auto eol=lf
```

**Comment by nathanfranke** [comment]
I can update the comment to be more clear. Something like:

# For addons, only export the addons folder. This is not needed for asset packs or projects.

**Comment by nathanfranke** [comment]
Updated to address Yuri's concern. Also changed `EOL` to `line endings` as it doesn't take much more space and is easier for people who don't understand what it means.

File: community/asset_library/submitting_to_assetlib.rst [hunk 1]
```diff
@@ -86,18 +86,13 @@ library a better place for all users.
...
-    # Ignore some files when exporting to a ZIP.
-    /.gitattributes     export-ignore
-    /.gitignore         export-ignore
-    /LICENSE            export-ignore
-    /LICENSE.md         export-ignore
-    /README.md          export-ignore
-    /project.godot      export-ignore
-    /icon.png           export-ignore
-    /icon.svg           export-ignore
+    # For addons, only include the addons folder. This is not needed for asset packs or projects.
```
Comment (aaronfranke) [inline community/asset_library/submitting_to_assetlib.rst:92]:
```suggestion
    # For addons, only include the addons folder when exporting a ZIP.
    # These lines should be deleted for asset packs or projects.
```

Comment (nathanfranke) [reply community/asset_library/submitting_to_assetlib.rst:92]: I don't think the average asset maker knows that the asset library uses GitHub ZIPs internally.

Comment (aaronfranke) [reply community/asset_library/submitting_to_assetlib.rst:92]: Attempt 2:

```suggestion
    # For addons, only include the addons folder when downloading from the Asset Library.
    # These lines should be deleted for asset packs or projects.
```

**Review by aaronfranke — APPROVED** [review]

**Comment by YuriSizov** [comment]
Not sure if this change addresses my concerns, but like I've said, I'm not blocking this in any way (hence why I didn't leave my concerns as a review). But if you are going to add extra instructions, you should do it in text, not in comments of code samples. Comments are not-translatable in our documentation, so this makes them hard to be useful for people reading documentation in languages other than English.

The old file also used comments, but those weren't really aimed at instructing the reader. This is, however:

> These lines should be deleted for asset packs or projects.

**Comment by nathanfranke** [comment]
Updated to use code tabs :slightly_smiling_face: 
image

**Comment by mhilbrunner** [comment]
Maybe @Calinou or someone else who spends more time than me around the Asset Lib can take a glance at this one too :)

**Comment by nathanfranke** [comment]
Should we change the tab to `Projects`?

**Comment by Calinou** [comment]
> Should we change the tab to `Projects`?

I would prefer changing it to "Projects / Templates / Demos" if we have the space, but "Projects" is OK too.

**Comment by nathanfranke** [comment]
> Asset packs should be distributed as add-ons, preferably in the `addons/` subfolder for organization purposes (although there is no hard rule).

Then let's be clear about it :slightly_smiling_face: 
image

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**